packages/terra-abstract-modal/src/AbstractModal.jsx

```diff
@@ -90,6 +94,7 @@ const defaultProps = {
   rootSelector: '#root',
   zIndex: '6000',
   isCalledFromNotificationDialog: false,
+  shouldTrapFocus: true,
```

Comment:
```suggestion
  shouldTrapFocus: false,
```
this should be false by default

Reply:
Updated.